app/controllers/images_controller.rb

```diff
@@ -160,8 +160,10 @@ def update_params
 
   def update_crop_params
     image_aspect_ratio = Image::HEIGHT.to_f / Image::WIDTH
-    crop_height = params[:crop_width].to_i * image_aspect_ratio
-    # FIXME: this will raise a warning because of unpermitted paramaters
-    params.permit(:crop_x, :crop_y, :crop_width).merge(crop_height: crop_height.to_i)
+
+    params
+      .require(:image_crop)
```

Comment:
Just a minor thing, but could this just be `image`, as on the edit page - it's just a bit redundant to have 'crop' twice :-).

Reply:
Sure, I think image_revision would be consistent.